packages/moleculer-mail/test/unit/template.spec.js

```diff
@@ -1,186 +0,0 @@
-"use strict";
-
-const { ServiceBroker } = require("moleculer");
-const MailService = require("../../src");
-const _ = require("lodash");
-
-describe("Test MailService template handling", () => {
-
```

Comment:
Why did you remove these files?

Reply:
I removed this file because it was testing the `getTemplate()` method which I removed.

Also, the rendering is not made by us anymore but by `email-templates` so testing the rendered email was not our concern anymore (IMO).

If you think of tests I missed or wrongfully removed, I'd be happy to write them back :smile: